wscript

```diff
@@ -479,6 +479,11 @@ video_output_features = [
         'desc': 'DRM',
         'deps': 'vt.h || consio.h',
         'func': check_pkg_config('libdrm', '>= 2.4.74'),
+    }, {
+        'name': '--drm-hdmi-hdr',
```

Comment:
Might want to grep through the PR and drop any HDMI references - things are very mixed at the moment.
Build toggle says "hdmi", runtime toggle does not etc.